Agents/FloodAssessmentAgent/README.md

```diff
@@ -181,21 +181,25 @@ To run the integration tests, access to the `docker.cmclinnovations.com` registr
 &nbsp;
 ## Dockerised agent integration tests
 
-The dockerised tests use one Docker container to initialise the Derivation agent and run pytest, and use Docker in Docker to spin up the required Blazegraph testcontainer instance:
+The dockerised tests use one Docker container to initialise the Derivation agent and run pytest, and use Docker in Docker to spin up the required Blazegraph testcontainer instance (depending on the system setup `docker compose` might need to be replaced with `docker-compose`):
 
 ```bash
 # Build and run Dockerised agent test
 docker compose -f "docker-compose-test_dockerised.yml" up -d --build
 ```
 
-Depending on the system setup `docker compose` might need to be replaced with `docker-compose`. In case of issues with starting the tests, try pulling the required Blazegraph image manually beforehand via `docker pull docker.cmclinnovations.com/blazegraph:1.0.0-SNAPSHOT`. 
-
 To run the dockerised tests in Debug mode, both a `docker-compose-test_dockerised_debug.yml` and a suitable `launch.json` configuration have been provided. Before starting to debug, please:
 
 - Copy the content of the `env_configs_mock.py` and `stack_configs_mock.py` files into the respective counterparts in the `floodassessment\utils\` repository (This is necessary as the agent codes and tests are mounted as volumes instead of being copied (and adjusted) into the container)
 - Adjust the import of `env_configs_mock` to `env_configs`
+- Uncomment the `affected_population = self.estimate_number_of_affected_people(warning_iri)` line (around line 172) within `impact_assessment.py` and replace with `affected_population = None`
 - Please also note that some issues with attaching the Debugger have been observed (i.e. `connect ECONNREFUSED 127.0.0.1:5678 `) using VS Code. In case the Debugger does not attach right away, try attaching using the `Python: Reattach and Debug` configuration after the test container has been started. 
 
+The tests have been successfully executed on both Windows and WSL; however, please note the following:
+- There seem to be internal communication issues when running the tests on Digital Ocean, causing an exception along the lines of ```Caused by: HttpException: -1 Unexpected error making the query: GET http://host.docker.internal:27149/blazegraph/namespace/kb/sparql?query=``` (this might be related to permission/priviliges issues)
```

Comment:
Could you please try adding the below line as suggested in https://github.com/moby/libnetwork/pull/2348#issuecomment-1177610499 and see if it works?
```yml
extra_hosts:
    - "host.docker.internal:host-gateway"
```

Reply:
> Could you please try adding the below line as suggested in [moby/libnetwork#2348 (comment)](https://github.com/moby/libnetwork/pull/2348#issuecomment-1177610499) and see if it works?
> 
> ```yaml
> extra_hosts:
>     - "host.docker.internal:host-gateway"
> ```

Nice one! That solves it. I included the changes in commit [61db754](https://github.com/cambridge-cares/TheWorldAvatar/pull/637/commits/61db75474794f31dac0a1200ddc63fa4163b4db5)